auditbeat/magefile.go

```diff
@@ -96,6 +97,20 @@ func Fields() error {
 	return mage.GenerateFieldsYAML("module")
 }
 
+// GoTestUnit executes the Go unit tests.
+// Use TEST_COVERAGE=true to enable code coverage profiling.
```

Comment:
Could we call this just `COVERAGE` and `RACE`?

Reply:
I'm thinking that they should both be prefixed in some way to avoid collisions and be self-describing. How prefixing any variables that control test parameters with `TEST_`? Like...

- `TEST_RACE`
- `TEST_COVERAGE`

Reply:
++